### PR TITLE
Update client and server to pass AioController filename

### PR DESCRIPTION
**Pull request by PWAlessi**
Update client and server to pass AioController filename

### CODE REVIEW

**Review by glimchb — COMMENTED** [review]
looks good. 
there is another location with `/tmp/aio_bdev_file` below

File: storage/server/backend.go [hunk 1]
```diff
@@ -233,7 +233,7 @@ func (s *server) AioControllerCreate(ctx context.Context, in *pb.AioControllerCr
 	params := BdevAioCreateParams{
 		Name:      in.GetDevice().GetName(),
 		BlockSize: 512,
-		Filename:  "/tmp/aio_bdev_file",
+		Filename:  in.GetDevice().GetFilename(),
```
Comment (glimchb) [inline storage/server/backend.go:236]: there is another location with `/tmp/aio_bdev_file` below

Comment (PWAlessi) [reply storage/server/backend.go:236]: 👍 fixed

**Review by glimchb — APPROVED** [review]
lgtm

**Pull request merged**